Readme.MD

```diff
@@ -146,6 +147,12 @@ at least allow the role to assume itself:
 }
 ```
 
+### Add config.properties parameters to support silent mode without prompt for user input (2018/6/22 update)
+- Add commandline support for ```--profile``` to specify OKTA_PROFILE
+- OKTA_AWS_REGION
+
+Same parameters can be set as environment variable. config.properties parameter has priority over environment variable.
```

Comment:
:bug: This is incorrect.

The environment variables override config.properties:
https://github.com/oktadeveloper/okta-aws-cli-assume-role/blob/master/src/main/java/com/okta/tools/OktaAwsConfig.java#L53-L57

This is intentional to allow quick overrides of behavior for a single shell session rather than editing the config file which represents the global defaults.

Reply:
I had just cut-n-pasted the line below which is also incorrect. https://github.com/oktadeveloper/okta-aws-cli-assume-role/blame/master/Readme.MD#L154

should I fix both?

Reply:
any thoughts on this?

Reply:
Good catch. Please fix both. Otherwise this is good to go. Thank you.